### DISCUSSION TITLE
Connection to Synology NAS not showing in dropdown menu

### DISCUSSION THREAD

**Discussion by rvmeush** [iterate-ch/cyberduck#15400] [Q&A]
I have a Synology NAS on my local network and have added it as a connection on Mountain Duck.  I can find the NAS under the Bonjour menu item, but the NAS doesn't appear as a device on the main menu for Mountain Duck.  I am not sure how this is supposed t work and would appreciate any input from someone who has added a Synology NAS as a connection.  Thanks.

**Comment by dkocher** [comment]
You can connect using the _Bonjour_ menu and then choose _New Bookmark_ [^1] from the context menu in Finder.app.

[^1]: https://docs.mountainduck.io/mountainduck/interface/#new-bookmark
